### ISSUE TITLE
NVAPI Error at launch. 

### ISSUE THREAD

**Issue by ahmlet** [Maassoft/ColorControl#23]
![ColorControl_TPGJzgk7bC](https://user-images.githubusercontent.com/71396599/113257392-ef2c1b80-92a0-11eb-90b4-63e6dcbc4481.png)

Got this error when launching.
My specs:
Ryzen 3600
asrock x470 master sli
2060 super
nvidia 465.89 drivers
4K TV lg uj6560
Log file
[LogFile.txt](https://github.com/Maassoft/ColorControl/files/6241704/LogFile.txt)

**Comment by Maassoft** [comment]
Thanks for the bug report. It seems reading the current color settings leads to an error somewhere in the Nvidia API. That's strange. Normally this will only occur when setting invalid values. It's probably not the driver because I'm using 465.89 as well.
What color settings are you using for these: desktop/output color depth, color format and dynamic range?

**Comment by ahmlet** [comment]
tried all combinations:
RGB FULL
Ycbcr422/8bit/10bit
Ycbcr444/8bit
HDR  ON/OFF wich each different settings
At one time I got a message from the TV that some remote wanted to connect, I dont know if it is the app.
I never once could manage to launch so I didnt change any of the deafult settings

**Comment by Maassoft** [comment]
Yes, I think the app wanted to connect to the TV. But regarding your error, I will add exception handling on reading the color data in the next version so that you at least should be able to start the application.

**Comment by ahmlet** [comment]
ok, thank you. looking forward

**Comment by ahmlet** [comment]
thanks, tried latest version and now its working fine. BTW is there a guide somewhere explaining about the different settings?
